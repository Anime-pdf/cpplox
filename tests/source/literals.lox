"string"
"multi
line
string"

214
214.12
24444.11

"unterminated string