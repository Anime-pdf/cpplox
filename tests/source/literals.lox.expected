[line 10] Error: Unterminated string.
STRING "string" string
STRING "multi
line
string" multi
line
string
NUMBER 214 214.000000
NUMBER 214.12 214.120000
NUMBER 24444.11 24444.110000
EOF  null